pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -182,7 +182,8 @@
      * 1 second.
      *
      * <p>By default, the acknowledge timeout is disabled and that means that messages delivered to a
-     * consumer will not be re-delivered unless the consumer crashes.
+     * consumer will not be re-delivered unless the consumer crashes. Since 2.3.0, when a dead letter policy
```

Comment:
```suggestion
     * consumer is not re-delivered unless the consumer crashes. Since 2.3.0, when a dead letter policy
```
Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

Reply:
Added second commit with these grammar changes ;)